2022-07-19 10:39:12,040 INFO    MainThread:2920 [wandb_setup.py:_flush():76] Configure stats pid to 2920
2022-07-19 10:39:12,040 INFO    MainThread:2920 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Nicor\.config\wandb\settings
2022-07-19 10:39:12,040 INFO    MainThread:2920 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Nicor\OneDrive\Documents\KolkowitzLab\Ellipse fitting\Learners\wandb\settings
2022-07-19 10:39:12,041 INFO    MainThread:2920 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-07-19 10:39:12,041 INFO    MainThread:2920 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Learner 1 wandb\\load_&_test_models.py', 'program': 'c:\\Users\\Nicor\\OneDrive\\Documents\\KolkowitzLab\\Ellipse fitting\\Learners\\Learner 1 wandb\\load_&_test_models.py'}
2022-07-19 10:39:12,046 INFO    MainThread:2920 [wandb_init.py:_log_setup():461] Logging user logs to C:\Users\Nicor\OneDrive\Documents\KolkowitzLab\Ellipse fitting\Learners\wandb\run-20220719_103912-3v3r7l97\logs\debug.log
2022-07-19 10:39:12,046 INFO    MainThread:2920 [wandb_init.py:_log_setup():462] Logging internal logs to C:\Users\Nicor\OneDrive\Documents\KolkowitzLab\Ellipse fitting\Learners\wandb\run-20220719_103912-3v3r7l97\logs\debug-internal.log
2022-07-19 10:39:12,046 INFO    MainThread:2920 [wandb_init.py:init():495] calling init triggers
2022-07-19 10:39:12,047 INFO    MainThread:2920 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2022-07-19 10:39:12,047 INFO    MainThread:2920 [wandb_init.py:init():548] starting backend
2022-07-19 10:39:12,047 INFO    MainThread:2920 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-07-19 10:39:12,051 INFO    MainThread:2920 [backend.py:ensure_launched():217] starting backend process...
2022-07-19 10:39:12,801 INFO    MainThread:2920 [backend.py:ensure_launched():222] started backend process with pid: 6384
2022-07-19 10:39:12,803 INFO    MainThread:2920 [wandb_init.py:init():558] backend started and connected
2022-07-19 10:39:12,806 INFO    MainThread:2920 [wandb_init.py:init():634] updated telemetry
2022-07-19 10:39:12,809 INFO    MainThread:2920 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-07-19 10:39:42,822 INFO    MainThread:2920 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2022-07-19 10:39:42,822 ERROR   MainThread:2920 [wandb_init.py:init():674] backend process timed out
2022-07-19 10:39:42,822 ERROR   MainThread:2920 [wandb_init.py:init():685] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-07-19 10:40:45,293 INFO    MainThread:2920 [wandb_init.py:teardown():302] tearing down wandb.init
